src/README.md

```diff
@@ -0,0 +1,47 @@
+## How to run?
+
+#### From scratch
+
+1. Download PrestaShop.
+
+```bash
+../scripts/download_prestashop.sh
```

Comment:
from which directory should i run it?

Reply:
You can run it from any directory - the path is relative to the script.